src/comments.test.js

```diff
@@ -0,0 +1,15 @@
+import { JSDOM } from 'jsdom';
+import { countElements } from './comments.js';
+
+const dom = new JSDOM(`<!DOCTYPE html><body><ul id="addList" class="flcol"></body>`);// eslint-disable-line
```

Comment:
- [ ] Kindly remove the eslint disable rule here. Its best to fix the issue in the code.